user-docs/collaboration-and-users/roles-and-permissions.md

```diff
@@ -45,6 +45,6 @@ Guest authors can be attributed to a page or post without having to register or
 
 ## Roles with plugins
 
-Some plugins can generate additional roles. For example, the SEO plugin Yoast comes with Altis. That’s why there are two extra roles from Yoast: [SEO Editor and SEO Manager](https://yoast.com/seo-roles/):
+Some plugins can generate additional roles. For example, the SEO plugin from Yoast. That’s why there are two extra roles from Yoast:
```

Comment:
Not sure I understand this. If Yoast is no longer bundled, should this say that there *may* be extra roles from plugins such as Yoast SEO?

Reply:
The text is referring to a screenshot which shows the extra roles. 
I may tweak the wording when I merge the update parts.